config/configuration.json

```diff
@@ -1073,6 +1081,20 @@
       "tags":"highway"
     }
   },
+  "StreetNameCheck": {
```

Comment:
Looks like your street name check commits ended up in this PR as well. Please clean that up so that this PR only contains changes for the highway access check.